vector-im/element-android
# [Devices Management] Session overview screen (PSG-691, PSG-693)

## Type of change


## Content


- Adding the new Session overview screen. In this screen, we only see a single view with session info for now. This new screen can be reached from a tap on current session info view (whole card or details button) and from any session item from other sessions list.
- Mutualizing the `SessionInfoView` which can be reused for current session or any session.
- Adding UseCases with tests to compute the trust level of a given Session.

## Motivation and context


Closes #6961 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/188402942-6a1e98e2-5108-481d-ae7c-13787d6ce1f2.gif" width=300 />

## Tests



- Enable the feature flag for new device management
- Go to Settings -> Security&Privacy -> Show all Sessions (V2, WIP)
- Press the current session view or the details button
- Check the Session overview screen is opened with the correct info corresponding to the current session
- Press back
- Press any session item from the "Other sessions" list
- Check the Session overview screen is opened with the correct info corresponding to the selected session

## Tested devices

- OS version(s): Android 11

## Checklist
